src/library/lib.rs

```diff
@@ -47,7 +47,8 @@ pub const LIB_ID_TAG: [u8; 32] = *b"urn:ubideco:aluvm:lib:v01#230304";
 #[derive(Wrapper, Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Hash, Default, Debug, Display, From)]
 #[wrapper(Deref, BorrowSlice, Hex, Index, RangeOps)]
 #[display(Self::to_baid58_string)]
-#[derive(StrictType, StrictEncode, StrictDecode)]
+#[derive(StrictType, StrictDecode)]
+#[cfg_attr(feature = "std", derive(StrictEncode))]
```

Comment:
Hm, is strict encoding not working with no_std? That is a bug, needs to be fixed...

What type of error you was getting when deriving StrictEncode under no_std?

Reply:
```rust
error[E0433]: failed to resolve: could not find `std` in the list of imported crates
  --> src/library/lib.rs:51:10
   |
51 | #[derive(StrictEncode)]
   |          ^^^^^^^^^^^^ could not find `std` in the list of imported crates
   |
   = note: this error originates in the derive macro `StrictEncode` (in Nightly builds, run with -Z macro-backtrace for more info)
```

not sure what is going on from the error output. I will investigate

Reply:
I think it is because we use `std::io` here https://github.com/6293/strict-encoding/blob/57f0cbc09edd23c896240913fb4c2639f030a34b/rust/src/traits.rs#L22

Reply:
Thank you, will address the issue there - and we will need to revert the change here after